docs/equality.md

```diff
@@ -78,18 +78,18 @@ Test the equality of structures nested within arrays:
 
 ```ts
 import { array, eq } from "fp-ts";
+import { Eq } from "fp-ts/lib/Eq";
 
 type Point = {
   x: number;
   y: number;
 };
 
-const eqPoint: eq.Eq<Point> = eq.getStructEq({
-  x: eq.eqNumber,
-  y: eq.eqNumber,
-});
+const eqPoint: Eq<Point> = {
```

Comment:
We can use https://gcanti.github.io/fp-ts/modules/Eq.ts.html#struct here.

Reply:
Fixed it.